vinz.user.js

```diff
@@ -41,37 +63,47 @@ $( document ).ready(function() {
     var window_url    = window.location.href;
     var change_number = window_url.split('/')[5];
     gerritGet("changes/" + change_number +  "/detail?O=404", function (data) {
-      $( "body" ).append("<h1/>").text("Change #"+ data._number + " " + data.subject + " -- " + data.owner.name);
+      $('#header').text("#"+ data._number + " ")
+        .append($("<small/>").text(data.subject))
+        .append($("<h3/>").text(data.owner.username)
+          .append($("<small/>").text(" " + data.owner.name))
+          .append( "<form class='form-inline pull-right'><div class='form-group'><label class='sr-only' for='vote'>Vote:</label><div class='btn-group' role='group'><button class='form-control btn btn-primary' type='button' id='vote'><span class='glyphicon glyphicon-thumbs-up'></span></button><button class='form-control btn btn-primary' type='button' id='vote'><span class='glyphicon glyphicon-thumbs-down'></span></button></div></form>" ));
+
+console.log(data);
+    var root_data = data;
 
-      $( "<p>").text("Files changed:").appendTo( "body" );
 
       // Create list of files that have been changed
       var get_url = "changes/" + data._number + "/revisions/" + data.current_revision + "/files";
       gerritGet( get_url, function (data) {
+console.log(data);
         var items = [];
+        var file_root = '/#/c/' + root_data._number + '/' + root_data.revisions[root_data.current_revision]._number + '/';
         $.each( data, function( index ) {
-          items.push( "<li id='" + index + "'>" + index + "</li>" );
+          items.push( "<a href='" + file_root + index + "' class='list-group-item' id='" + index + "'>" + index + "</a>" );
         });
-        $( "<ul/>", {
-          "class": "files-modified-list",
+        $( "<div/>", {
+          "class": "files-modified-list list-group",
           html: items.join( "" )
-        }).appendTo( "body" );
+        }).appendTo($files);
+console.log('remove the styles');
+        setTimeout(function() { $('style').remove(); }, 5000);
```

Comment:
There is something funky going on that requires this line... still need to debug that.